isis/src/tgo/tsts/ingestReingest/Makefile

```diff
@@ -0,0 +1,24 @@
+APPNAME =
```

Comment:
Suggestion to either use this `APPNAME` variable for `tgocassis2isis` or get rid of it. 

You don't need to change this, but I did like the approach you used in the coloredMosaic cat test you wrote that handled this as follows: 

```
INGEST = tgocassis2isis
SPICE = spiceinit
MAPFILE = mosrange
PROJECT = cam2map
MOSAIC = automos
EXPORT = isis2pds
GETKEY = getkey
CUBEIT = cubeit
```